### ISSUE TITLE
Compile error.

### ISSUE THREAD

**Issue by BoJack62** [alexchang0229/SatelliteTracker#4]
I receive a compile error “ ‘passinfo’ not declared in this scope” when attempting to compile using either Arduino IDE or Web Editor.

**Comment by alexchang0229** [comment]
Hi Bojack,

Sorry for the late response, for some reason I didn't get notified.

Just so I understand your problem, are you compiling 'tracker_main.ino' and it's giving you an error? 

Could you post a screenshot of your compiler error?

Thanks!

**Comment by BoJack62** [comment]
Alex,
  Thanks for getting back to me.
  It’s been years since I’ve worked with C.  There was a problem with the Predict function, “min” variable I believe.  I contacted someone else that had the function posted and they corrected the problem.
  Then, I wasn’t able to compile using tabs for different functions in Arduino Editor.  Never did figure it out, I managed with some experimentation as to their order, to cut and past all functions into one file and then successfully compile.
  Next issue arose when I started up the Arduino on tracker.  I could not tell wether it was on and functioning or not, so I added an on board LED blink to code. Still nothing with just battery.
  Connected USB cable, and it ran fine.
  I had to remark out the while !serial lines to allow the program to continue if there is no serial connection.  Now it blinks when on and I have successfully tested it.
  One question,  is the initial EL 20 or 25 degrees?  I seem to remember both being listed.
  [Image.jpeg] [Image.jpeg]
Thanks,
Bo Jack Murdock
________________________________

**Comment by alexchang0229** [comment]
Yeah, I had a few people contact me about the 'min' variable, that's on me, I guess it's a special word in C++. I Changed that in the most recent update. Just needed to change the name of the variable where it appears.

As for compiling with the functions in different files, as long as all the .ino files are in the same folder named '/tracker_main' it should work; I also updated the repository to reflect this. 

And at [line 11](https://github.com/alexchang0229/SatelliteTracker/blob/de7fbdbdac04e80347da736d1a5495d760cc3895/tracker_main/tracker_main.ino#L11) there's `#define DEBUG` which is used in [line 86](https://github.com/alexchang0229/SatelliteTracker/blob/de7fbdbdac04e80347da736d1a5495d760cc3895/tracker_main/tracker_main.ino#L86) in `#ifdef DEBUG`. If DEBUG is defined, the Arduino will wait for a serial connection so it can print out what it's doing. If you don't need that you can get rid of the `#define DEBUG` line.

The initial EL should be 20 degrees, this is defined in [line 262](https://github.com/alexchang0229/SatelliteTracker/blob/de7fbdbdac04e80347da736d1a5495d760cc3895/tracker_main/tracker_main.ino#L262).  The [motors I'm using](https://www.amazon.com/Longruner-Stepper-Uln2003-arduino-LK67/dp/B015RQ97W8) have a step angle of  5.625 x 1/64 so at 227 steps it's around 20 degrees.

Hopes this helps, let me know if you have any other questions!

Alex
